MoltenVK/MoltenVK/Utility/MVKConfigMembers.def

```diff
@@ -81,6 +81,8 @@ MVK_CONFIG_MEMBER(useMetalArgumentBuffers,                MVKUseMetalArgumentBuf
 MVK_CONFIG_MEMBER(shaderSourceCompressionAlgorithm,       MVKConfigCompressionAlgorithm,            SHADER_COMPRESSION_ALGORITHM)
 MVK_CONFIG_MEMBER(shouldMaximizeConcurrentCompilation,    VkBool32,                                 SHOULD_MAXIMIZE_CONCURRENT_COMPILATION)
 MVK_CONFIG_MEMBER(timestampPeriodLowPassAlpha,            float,                                    TIMESTAMP_PERIOD_LOWPASS_ALPHA)
+MVK_CONFIG_MEMBER(useMetalPrivateAPI,                     VkBool32,                                 USE_METAL_PRIVATE_API)
+MVK_CONFIG_MEMBER(_reserved,                              uint32_t,                                 USE_METAL_PRIVATE_API)
```

Comment:
This padding member will need a unique name. Per my note above, perhaps something like:

    MVK_CONFIG_MEMBER(_unused_struct_padding,                              uint32_t,                                 UNUSED_STRUCT_PADDING)